winlogbeat/beater/winlogbeat.go

```diff
@@ -231,17 +232,17 @@ loop:
 		// Publish events.
 		numEvents := int64(len(events))
 		ok := eb.client.PublishEvents(events, publisher.Sync, publisher.Guaranteed)
-		if ok {
-			publishedEvents.Add("total", numEvents)
-			publishedEvents.Add(api.Name(), numEvents)
-			logp.Info("EventLog[%s] Successfully published %d events",
-				api.Name(), numEvents)
-		} else {
-			logp.Warn("EventLog[%s] Failed to publish %d events",
-				api.Name(), numEvents)
-			publishedEvents.Add("failures", 1)
```

Comment:
The changes look good. Since "failures" is never used (line 137), let's remove it.